<!DOCTYPE html PUBLIC '-//W3C//DTD XHTML 1.0 Transitional//EN' 'http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd'>
<html xmlns='http://www.w3.org/1999/xhtml'>
<head>
<title>exception</title>
<meta name='robots' content='noindex,nofollow' />
<meta name='generator' content='GLOBAL-6.5.6' />
<meta http-equiv='Content-Style-Type' content='text/css' />
<link rel='stylesheet' type='text/css' href='../style.css' />
</head>
<body>
<pre>
<span class='curline'><a href='../S/1097.html#L526'>exception</a>         526 src/config.hpp 	struct error : public game::error, public boost::exception {</span>
<span class='curline'><a href='../S/1077.html#L29'>exception</a>          29 src/exceptions.hpp struct error : std::exception</span>
<span class='curline'><a href='../S/314.html#L785'>exception</a>         785 src/filesystem_boost.cpp 	catch(const std::exception ex)</span>
<span class='curline'><a href='../S/1080.html#L44'>exception</a>          44 src/game_end_exceptions.hpp class return_to_play_side_exception : public lua_jailbreak_exception, public std::exception</span>
<span class='curline'><a href='../S/1080.html#L50'>exception</a>          50 src/game_end_exceptions.hpp 		, std::exception()</span>
<span class='curline'><a href='../S/1080.html#L61'>exception</a>          61 src/game_end_exceptions.hpp 	, public std::exception</span>
<span class='curline'><a href='../S/745.html#L446'>exception</a>         446 src/game_launcher.cpp 		} catch (std::exception &amp; e) {</span>
<span class='curline'><a href='../S/844.html#L80'>exception</a>          80 src/gettext_boost.cpp 			catch(const std::exception&amp; e)</span>
<span class='curline'><a href='../S/520.html#L32'>exception</a>          32 src/gui/core/layout_exception.hpp struct layout_exception_width_modified : public std::exception</span>
<span class='curline'><a href='../S/520.html#L37'>exception</a>          37 src/gui/core/layout_exception.hpp struct layout_exception_resize_failed : public std::exception</span>
<span class='curline'><a href='../S/1098.html#L541'>exception</a>         541 src/halo.cpp   		} catch (std::exception &amp; e) {</span>
<span class='curline'><a href='../S/684.html#L117'>exception</a>         117 src/lexical_cast.hpp struct bad_lexical_cast : std::exception</span>
<span class='curline'><a href='../S/1106.html#L18'>exception</a>          18 src/libc_error.hpp class libc_error : public std::exception</span>
<span class='curline'><a href='../S/944.html#L93'>exception</a>          93 src/lua_jailbreak_exception.hpp 		type exception(dynamic_cast&lt;type&amp;&gt;(*jailbreak_exception));   \</span>
<span class='curline'><a href='../S/944.html#L94'>exception</a>          94 src/lua_jailbreak_exception.hpp 		throw exception;                                             \</span>
<span class='curline'><a href='../S/843.html#L35'>exception</a>          35 src/save_blocker.cpp 	} catch (std::exception &amp; e) {</span>
<span class='curline'><a href='../S/824.html#L77'>exception</a>          77 src/savegame.hpp 	: public lua_jailbreak_exception, public std::exception</span>
<span class='curline'><a href='../S/205.html#L246'>exception</a>         246 src/scripting/lua_fileops.cpp 	catch(const std::exception &amp; ex)</span>
<span class='curline'><a href='../S/972.html#L32'>exception</a>          32 src/sdl/exception.cpp exception::exception(const std::string&amp; operation, const bool use_sdl_error)</span>
<span class='curline'><a href='../S/976.html#L38'>exception</a>          38 src/sdl/exception.hpp 	exception(const std::string&amp; operation, const bool use_sdl_error);</span>
<span class='curline'><a href='../S/975.html#L36'>exception</a>          36 src/sdl/window.cpp 		throw exception("Failed to create a SDL_Window object.", true);</span>
<span class='curline'><a href='../S/975.html#L40'>exception</a>          40 src/sdl/window.cpp 		throw exception("Failed to create a SDL_Renderer object.", true);</span>
<span class='curline'><a href='../S/975.html#L45'>exception</a>          45 src/sdl/window.cpp 		throw exception("Failed to retrieve the information of the renderer.",</span>
<span class='curline'><a href='../S/975.html#L50'>exception</a>          50 src/sdl/window.cpp 		throw exception("The renderer has no texture information available.\n",</span>
<span class='curline'><a href='../S/975.html#L102'>exception</a>         102 src/sdl/window.cpp 		throw exception("Failed to clear the SDL_Renderer object.",</span>
<span class='curline'><a href='../S/124.html#L161'>exception</a>         161 src/serialization/schema_validator.cpp 			catch (std::exception){</span>
<span class='curline'><a href='../S/128.html#L38'>exception</a>          38 src/serialization/unicode_types.hpp 	class invalid_utf8_exception : public std::exception {};</span>
<span class='curline'><a href='../S/705.html#L206'>exception</a>         206 src/tests/gui/test_gui2.cpp 			std::string exception;</span>
<span class='curline'><a href='../S/705.html#L210'>exception</a>         210 src/tests/gui/test_gui2.cpp 				exception = "gui2::layout_exception_width_modified";</span>
<span class='curline'><a href='../S/705.html#L212'>exception</a>         212 src/tests/gui/test_gui2.cpp 				exception = "gui2::layout_exception_width_resize_failed";</span>
<span class='curline'><a href='../S/705.html#L214'>exception</a>         214 src/tests/gui/test_gui2.cpp 				exception = "gui2::layout_exception_height_resize_failed";</span>
<span class='curline'><a href='../S/705.html#L216'>exception</a>         216 src/tests/gui/test_gui2.cpp 				exception = e.dev_message;</span>
<span class='curline'><a href='../S/705.html#L217'>exception</a>         217 src/tests/gui/test_gui2.cpp 			} catch(std::exception&amp; e) {</span>
<span class='curline'><a href='../S/705.html#L218'>exception</a>         218 src/tests/gui/test_gui2.cpp 				exception = e.what();</span>
<span class='curline'><a href='../S/705.html#L220'>exception</a>         220 src/tests/gui/test_gui2.cpp 				exception = "unknown";</span>
<span class='curline'><a href='../S/705.html#L222'>exception</a>         222 src/tests/gui/test_gui2.cpp 			BOOST_CHECK_MESSAGE(exception.empty(),</span>
<span class='curline'><a href='../S/705.html#L227'>exception</a>         227 src/tests/gui/test_gui2.cpp 					&lt;&lt; "\nException caught: " &lt;&lt; exception &lt;&lt; '.');</span>
<span class='curline'><a href='../S/705.html#L245'>exception</a>         245 src/tests/gui/test_gui2.cpp 				std::string exception;</span>
<span class='curline'><a href='../S/705.html#L252'>exception</a>         252 src/tests/gui/test_gui2.cpp 					exception = "gui2::layout_exception_width_modified";</span>
<span class='curline'><a href='../S/705.html#L254'>exception</a>         254 src/tests/gui/test_gui2.cpp 					exception = "gui2::layout_exception_width_resize_failed";</span>
<span class='curline'><a href='../S/705.html#L256'>exception</a>         256 src/tests/gui/test_gui2.cpp 					exception = "gui2::layout_exception_height_resize_failed";</span>
<span class='curline'><a href='../S/705.html#L258'>exception</a>         258 src/tests/gui/test_gui2.cpp 					exception = e.dev_message;</span>
<span class='curline'><a href='../S/705.html#L259'>exception</a>         259 src/tests/gui/test_gui2.cpp 				} catch(std::exception&amp; e) {</span>
<span class='curline'><a href='../S/705.html#L260'>exception</a>         260 src/tests/gui/test_gui2.cpp 					exception = e.what();</span>
<span class='curline'><a href='../S/705.html#L262'>exception</a>         262 src/tests/gui/test_gui2.cpp 					exception = "unknown";</span>
<span class='curline'><a href='../S/705.html#L264'>exception</a>         264 src/tests/gui/test_gui2.cpp 				BOOST_CHECK_MESSAGE(exception.empty(),</span>
<span class='curline'><a href='../S/705.html#L269'>exception</a>         269 src/tests/gui/test_gui2.cpp 						&lt;&lt; "\nException caught: " &lt;&lt; exception &lt;&lt; '.');</span>
<span class='curline'><a href='../S/705.html#L291'>exception</a>         291 src/tests/gui/test_gui2.cpp 			std::string exception;</span>
<span class='curline'><a href='../S/705.html#L299'>exception</a>         299 src/tests/gui/test_gui2.cpp 				exception = "gui2::layout_exception_width_modified";</span>
<span class='curline'><a href='../S/705.html#L301'>exception</a>         301 src/tests/gui/test_gui2.cpp 				exception = "gui2::layout_exception_width_resize_failed";</span>
<span class='curline'><a href='../S/705.html#L303'>exception</a>         303 src/tests/gui/test_gui2.cpp 				exception = "gui2::layout_exception_height_resize_failed";</span>
<span class='curline'><a href='../S/705.html#L305'>exception</a>         305 src/tests/gui/test_gui2.cpp 				exception = e.dev_message;</span>
<span class='curline'><a href='../S/705.html#L306'>exception</a>         306 src/tests/gui/test_gui2.cpp 			} catch(std::exception&amp; e) {</span>
<span class='curline'><a href='../S/705.html#L307'>exception</a>         307 src/tests/gui/test_gui2.cpp 				exception = e.what();</span>
<span class='curline'><a href='../S/705.html#L309'>exception</a>         309 src/tests/gui/test_gui2.cpp 				exception = "unknown";</span>
<span class='curline'><a href='../S/705.html#L311'>exception</a>         311 src/tests/gui/test_gui2.cpp 			BOOST_CHECK_MESSAGE(exception.empty(),</span>
<span class='curline'><a href='../S/705.html#L316'>exception</a>         316 src/tests/gui/test_gui2.cpp 					&lt;&lt; "\nException caught: " &lt;&lt; exception &lt;&lt; '.');</span>
<span class='curline'><a href='../S/1008.html#L723'>exception</a>         723 src/units/unit.cpp 	} catch(std::exception &amp; e) {</span>
<span class='curline'><a href='../S/861.html#L93'>exception</a>          93 src/utils/make_enum.hpp class bad_enum_cast : public std::exception</span>
<span class='curline'><a href='../S/866.html#L23'>exception</a>          23 src/utils/name_generator.hpp class name_generator_invalid_exception : public std::exception {</span>
<span class='curline'><a href='../S/1099.html#L25'>exception</a>          25 src/variable_info.hpp class invalid_variablename_exception : public std::exception</span>
<span class='curline'><a href='../S/1099.html#L28'>exception</a>          28 src/variable_info.hpp 	invalid_variablename_exception() : std::exception() {}</span>
<span class='curline'><a href='../S/114.html#L1091'>exception</a>        1091 src/wesnoth.cpp 	} catch(const sdl::exception&amp; e) {</span>
<span class='curline'><a href='../S/114.html#L1101'>exception</a>        1101 src/wesnoth.cpp 	} catch(std::exception &amp; e) {</span>
<span class='curline'><a href='../S/946.html#L93'>exception</a>          93 src/wesnoth_lua_config.h 		} catch(const std::exception &amp;e) { \</span>
<span class='curline'><a href='../S/1573.html#L43'>exception</a>          43 utils/umc_dev/org.wesnoth/src-gen/org/wesnoth/wml/impl/WmlFactoryImpl.java     catch (Exception exception)</span>
<span class='curline'><a href='../S/1573.html#L45'>exception</a>          45 utils/umc_dev/org.wesnoth/src-gen/org/wesnoth/wml/impl/WmlFactoryImpl.java       EcorePlugin.INSTANCE.log(exception);</span>
</pre>
</body>
</html>
